roles/datadog-builder/tasks/datadog-builder.yml

```diff
@@ -64,3 +73,4 @@
       /opt/venvs/datadog-builder/bin/datadog-builder
       --config /etc/datadog-builder/datadog-secrets.yml
       update {{ datadog_builder_monitor_dir }}/{{ datadog_builder_monitor_file }}
+      2>> /var/log/datadog-builder.log > /dev/null
```

Comment:
why is stdout going to /dev/null? is it not useful? 

If it's not we can do things like the standard python logging configuration in datadog-builder and make this more sane.